src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -271,13 +273,27 @@ export const handleGetPublicForm: ControllerHandler<
     case AuthType.SP:
     case AuthType.CP:
       return SpcpService.extractJwtPayloadFromRequest(authType, req.cookies)
-        .map(({ userName }) =>
-          res.json({
+        .map(({ userName, exp, iat, rememberMe }) => {
+          if (!exp) {
+            {
+              logger.error({
+                message: 'Invalid expiry time for cookie',
+                meta: logMeta,
+              })
+            }
```

Comment:
think I left a comment abt this previously but why is there an extra pair of braces here?

Reply:
thanks! sorry left it out when i reinstated the commit. it's been removed